src/main/webapp/app/entities/programming-exercise-test-case.model.ts

```diff
@@ -25,4 +26,8 @@ export class ProgrammingExerciseTestCase implements BaseEntity {
     exercise?: ProgrammingExercise;
     type?: ProgrammingExerciseTestCaseType;
     solutionEntries?: ProgrammingExerciseSolutionEntry[];
+    resultingPoints?: number;
+    resultingPointsPercent?: number;
+    testCaseStats?: TestCaseStats;
+    changed?: boolean;
```

Comment:
They are not part of the actual object that is also stored on the server, are they?
Maybe add an empty line above them and maybe even some comment about them

Reply:
Done